src/main.js

```diff
@@ -32,19 +31,19 @@
   var ffts = initializeFFTs(20, bufferSize);
   var buffer = null;
 
-  var renderer = new THREE.WebGLRenderer();
-  renderer.setPixelRatio(window.devicePixelRatio ? window.devicePixelRatio : 1);
-  renderer.setSize(resolution * aspectRatio, resolution);
-  renderer.domElement.style.width = '100%';
-  renderer.domElement.style.height = 'auto';
-  // window.addEventListener('resize', function(){
-  //   let canvasWidth = document.querySelector('.col').offsetWidth;
-  //   resolution = canvasWidth/16*10;
-  //   renderer.setSize(resolution * aspectRatio, resolution);
-  //   renderer.domElement.height = resolution * window.devicePixelRatio;
-  //   renderer.domElement.width = canvasWidth * window.devicePixelRatio;
-  // });
-  document.querySelector('#showcase').appendChild(renderer.domElement);
+  var renderer = new THREE.WebGLRenderer({ canvas: document.querySelector("canvas") });
+
+
+  function resize() {
+    var resolution = renderer.domElement.parentNode.offsetWidth / 16 * 10;
+    renderer.setPixelRatio(window.devicePixelRatio ? window.devicePixelRatio : 1);
+    renderer.setSize(resolution * aspectRatio, resolution);
+    renderer.domElement.style.width = '100%';
```

Comment:
from: https://threejs.org/docs/manual/introduction/Creating-a-scene.html

> If you wish to keep the size of your app but render it at a lower resolution, you can do so by calling setSize with false as updateStyle (the third argument). For example, setSize(window.innerWidth/2, window.innerHeight/2, false) will render your app at half resolution, given that your <canvas> has 100% width and height

Does that help?